README.md

```diff
@@ -77,60 +77,139 @@ Notes:
+**Note:** The version numbers of the latest files will be higher than this example, but the naming format will be the same.  
 
-* seedsigner_pubkey.gpg (from the main folder of this repo)
-* seedsigner_0_4_6.img.zip (from the software release)
-* seedsigner_0_4_6.img.zip.sha256 (from the software release)
-* seedsigner_0_4_6.img.zip.sha256.sig (from the software release)
 
-**Note:** The specific version number of the files in your folder might not match the above exactly, but their overall format and amount should be the same.
+## Verifying that the downloaded files are authentic (optional but highly recommended!)
 
-This process also assumes you are running the commands from a system where both [GPG](https://gnupg.org/download/index.html) and [shasum](https://command-not-found.com/shasum) are installed and working.
+You can quickly verify that the downloaded software is both authentic and unaltered, by following these instructions.
+
+These next steps assume you are running the commands from a computer where both [GPG](https://gnupg.org/download/index.html) and [shasum](https://command-not-found.com/shasum) are already installed, and that you also know [how to navigate on a terminal](https://terminalcheatsheet.com/guides/navigate-terminal). 
+
+
+### 1. Import the public key of the SeedSigner project into your computer
+
+The *fetch-keys*  command below will import the SeedSigner projects public key from a popular online keyserver called *Keybase.io*, into your computers *keychain*. 
+
+To begin, you will run the *fetch-keys* command as shown below (run it from inside the *same folder* that you saved the downloaded files into). 
 
-First make sure that the public key is present in your keychain:
-```
-gpg --import seedsigner_pubkey.gpg 
-```
-This command will import the public key, or return:
 ```
-key <...> not changed
+gpg --fetch-keys https://keybase.io/SeedSigner/pgp_keys.asc
 ```
+When the command completes successfully, it will show that this key was either imported or updated from Keybase.io. A numeric ID, as circled in red in the example below is known as the keys fingerprint. Please ignore the email address shown, because it is not part of any verification. 
+
+![SS - Keybase PubKey import with Fingerprint shown (New import or update of the key)](https://user-images.githubusercontent.com/91296549/174248861-7961c038-1fbf-47a1-a110-146cb218b1c8.jpg)  
+
+<details><summary>Learn more about how keybase.io helps you check that someone is who they say they are</summary>
+<p>
+  The Keybase.io website allows you to independently verify that the public key provided is authentic and that it belongs to the organization it claims to represent. 
+  Keybase has checked the pubkey cryptographically when it was saved in the 3 separate online locations. These are: on www.twitter.com/seedsigner, on the website www.seedsigner.com , and in the software repository at Github www.github.com/seedsigner.
```

Comment:
(see comment below)

Again, I'd reframe this as: You can independently verify the public key at these three links.

No need to trust Keybase's verification/aggregation. We can verify ourselves straight from each source.

Keybase is a convenience, but we could write a version of these docs that make no mention whatsoever of Keybase, right?

Reply:
This section has now been re-stated to describe both options: verifying via Keybase, and also verifying out-of-band without keybase at all. Mention is made to use the out-of-band method if keybase is not available or it no longer exists.